crates/typed-store/src/rocks/mod.rs

```diff
@@ -459,6 +450,8 @@ pub fn open_cf_opts<P: AsRef<Path>>(
     let primary = path.as_ref().to_path_buf();
 
     let rocksdb = {
+        let _span =
+            tracing::debug_span!("rocksdb::DBWithThreadMode::<MultiThreaded>::open_cf_descriptors");
```

Comment:
What is this supposed to do?  The span is not used anywhere.  If the intention is just to log instead of trace a span of code, let's use `tracing::trace!` instead.

Reply:
I didn't add or change this code, this was just `cargo fmt`

Reply:
I think this code was added in this PR, I don't see any corresponding '-' lines which you'd expect to see from cargo fmt.

Reply:
hmm you're right. my bad again, i started this change in a dirty tree by accident...